lib/resources/postgres.rb

```diff
@@ -81,6 +77,16 @@ def verify_dirs
       end
     end
 
+    def version_from_psql
```

Comment:
Ok. I was just following the previous convention from the code before I modified it.  If you think the warn statements are no longer needed then I am fine with removing them. I just left them because @arlimus had them there before.